gtsfm/frontend/detector_descriptor/aslfeat.py

```diff
@@ -0,0 +1,95 @@
+"""Implements the ASLFeat detector-descriptor by wrapping around the original Pytorch source code.
+
+Paper: https://arxiv.org/abs/1905.03561
+https://github.com/mihaidusmanu/d2-net
+
+Authors: Travis Driver
+"""
+
+from pathlib import Path
+from typing import Tuple, Dict, Any
+import sys
+
+import numpy as np
+import tensorflow.compat.v1 as tf
+
+import gtsfm.utils.images as image_utils
+from gtsfm.common.image import Image
+from gtsfm.common.keypoints import Keypoints
+from gtsfm.frontend.detector_descriptor.detector_descriptor_base import DetectorDescriptorBase
```

Comment:
do we do the from * imports first or the import * imports first? Looks like youve done both, "from imports" first for lines 9-11 and "import" imports first for 16-19.